src/app/views/collections/create/CreateNewCollection.jsx

```diff
@@ -146,6 +146,17 @@ const CreateNewCollection = props => {
         return new Date(`${newCollection.publishDate.value} ${newCollection.publishTime.value}`).toISOString();
     };
 
+    const makeTeams = () => {
+        if (isNewSignIn) {
+            return newCollection.teams.map(team => {
+                return team.id;
+            });
+        }
+        return newCollection.teams.map(team => {
+            return team.name;
+        });
+    };
```

Comment:
Would the following be simpler?
```suggestion
        return newCollection.teams.map(team => {
            if (isNewSignIn) {
                return team.id;
            } else {
                return team.name;
            }
        });
    };
```

Reply:
Also, would it be possible to add unit test coverage for the new sign in?

Reply:
I've added some unit tests for my change